istioctl/cmd/istioctl/convert/testdata/v1alpha3/destination-rule-helloworld-with-80-20.yaml.golden

```diff
@@ -0,0 +1,48 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  creationTimestamp: null
+  name: helloworld-service
+  namespace: default
+spec:
+  host: helloworld-service
+  subsets:
+  - labels:
+      version: "1.0"
+    name: version-1-0
+  - labels:
+      version: "2.0"
+    name: version-2-0
+  trafficPolicy:
+    connectionPool:
```

Comment:
I don't think this is quite right... `v1alpha1/destination-rule-helloworld.yaml` has a destination of `helloworld-service` `version:v1`. So the policy is supposed to only apply to the `version:v1` subset. This is applying the `v1alpha1/destination-rule-helloworld.yaml` policy globally to the `helloworld-service`. See https://github.com/istio/api/blob/master/networking/v1alpha3/destination_rule.proto#L59 for an example of a traffic policy being applied to a specific subset.